### PR TITLE
Update DataSetToDataRepositoryMapper.cs

### LINKED ISSUE #1090: LLOQ values from column are not used although configured
PKSim Version from 15.6.2021
Import attached file.
Filter for molecules GLP-1_7...  and Insulin and load and import
For the concentration values is the LLOQ value given in the concentration_LLOQ column:
for first molecule LLOQ = 15 , for the second molecule LLOQ = 100.
But not in the preview nor in the import results in PKSim the LLOQ is used.

I would expect display of LLOQ in the preview both in the table right of Error and in the chart.
[CSV JJ.xlsx](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/files/6662781/CSV.JJ.xlsx)
LLOQnotUsed

### PR DESCRIPTION
**Pull request by abdelr**
Fixes #1090 (Show LLOQ)

### CODE REVIEW

File: src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs [hunk 1]
```diff
@@ -105,6 +108,8 @@ private void convertParsedDataColumn(DataRepository dataRepository, KeyValuePair
             else
                values[i++] = (float) adjustedValue;
          }
+         if (lloqValue != null)
+            dataInfo.LLOQ = Convert.ToSingle(dimension.UnitValueToBaseUnitValue(dimension.FindUnit(lloqValue.Unit), lloqValue.Lloq));
```
Comment (msevestre) [inline src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs:112]: I think this should really be tested. A core function, no UI.

**Review by msevestre — COMMENTED** [review]
Ok to merge but I'd like to see tests for this

File: src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs (hunk 1 above)
Comment (abdelr) [reply src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs:112]: I agree... I will add some test on next week

Comment (abdelr) [reply src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs:112]: I fixed the ignored tests for the mapper and added a new one for the lloq

**Review by msevestre — APPROVED** [review]
Looks. Good. if possible, let's use the BDDHelper methods next time (ShouldBeEqualTo etc..)

**Pull request merged**